src/logic/safe/store/actions/transactions/fetchTransactions/loadModuleTransactions.ts

```diff
@@ -0,0 +1,33 @@
+import fetchTransactions from 'src/logic/safe/store/actions/transactions/fetchTransactions/fetchTransactions'
+import { TransactionTypes } from 'src/logic/safe/store/models/types/transaction'
+import { DataDecoded, Operation } from 'src/logic/safe/store/models/types/transactions.d'
+
+export type ModuleTxServiceModel = {
+  created: string
+  executionDate: string
+  blockNumber: number
+  transactionHash: string
+  safe: string
+  module: string
+  to: string
+  value: string
+  data: string
+  operation: Operation
+  dataDecoded: DataDecoded
+}
+
+let previousETag = null
```

Comment:
Why do we need to keep track of a etag? I don't see it's being used anywhere except the assignment.

Reply:
This should work as it works with the other requests.

`fetchTransactions` firstly uses `previousETag === null`, then the server returns the `eTag`, which is stored in this variable for the following requests.

On line `30` you can see the assignment `previousETag = eTag`